xc/xenops_server_xen.ml

```diff
@@ -2622,14 +2623,22 @@ module VBD = struct
              let open Device_common in
              let device_number = dev.frontend.devid |> Device_number.of_xenstore_key in
              let qemu_domid = this_domid ~xs in
-             let qemu_frontend = match Device_number.spec device_number with
-               | Ide, n, _ when n < 4 ->
+             let qemu_frontend =
+               let maybe_create_vbd_backend () =
```

Comment:
create vbd frontend for backend? Looking at the code below it creates a frontend, not a backend, based on the presence of a backend.

Reply:
oops